typhoeus.gemspec

```diff
@@ -19,7 +19,10 @@ Gem::Specification.new do |s|
 
   s.add_dependency('ethon', [">= 0.9.0"])
 
-  s.files        = `git ls-files`.split("\n")
-  s.test_files   = `git ls-files -- spec/*`.split("\n")
+  s.files = Dir.chdir(__dir__) do
+    `git ls-files -z`.split("\x0").reject do |file|
+      file.start_with?(*%w[. CONTRIBUTING Gemfile Guardfile Rakefile perf spec typhoeus.gemspec])
```

Comment:
Same as in ethon. The `typhoeus.gemspec` should be part of the package.

Reply:
Updated in [1dd91a1](https://github.com/typhoeus/typhoeus/pull/712/commits/1dd91a11e3c017fe5d8ae1963c0b13679d1d9b6d).